clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/handler/KubernetesDeploymentHandler.java

```diff
@@ -82,7 +86,7 @@ public KubernetesKind kind() {
 
   @Override
   public boolean versioned() {
-    return false;
+    return true;
```

Comment:
This change breaks a bunch of integration tests (which unfortunately weren't required to pass before merging this PR), but also is a pretty massive behavior change.  Before people who deploy a deployment named `foo`, would get `foo`.  Now they get `foo-vNNN`.  It also makes https://spinnaker.io/docs/reference/providers/kubernetes-v2/#workloads incorrect for Deployments.  I'm not sure how critical this change is to this PR....

Reply:
PR to revert this one: https://github.com/spinnaker/clouddriver/pull/5843